app/models/experiment.rb

```diff
@@ -217,14 +218,14 @@ def generate_workflow_img
                          use: :neato)
 
     graph[:size] = '4,4'
-    graph.node[color: '#d2d2d2',
+    graph.node[color: Constants::COLOR_ALTO,
```

Comment:
This line of code throws:
 Invalid value `#555` for attribute `fontcolor` : Wrong color definition RGBA #555 
error. Please fix that and check others